docs/xjit.md

```diff
@@ -67,6 +68,16 @@ These parameters can be used to modify the behavior of `-Xjit`:
 
 : From z/OS V2R3, residency mode for 64-bit applications (RMODE64) is enabled by default. This feature allows the JIT to allocate executable code caches above the 2 GB memory bar, which is the default behavior. Use this option to turn off this JIT behavior.
 
+### `enableGPU`
+
+**(Windows (x86-64) or Linux (x86-64 and IBM POWER 8))**
+
+        -Xjit:enableGPU
+
+: Enables the JIT compiler to offload certain processing tasks to a graphics processing unit (GPU). The JIT determines which functions to offload based on performance heuristics. Systems must support NVIDIA Compute Unified Device Architecture (CUDA).  GPU adapters require the CUDA Toolkit 7.5 and must have a minimum compute capability of 3.0.
```

Comment:
It isn't that adapters require CUDA 7.5, but rather, that JIT requires 7.5.

Reply:
Fixed. Same in the AdoptOpenJDK topic.